e2e-tests/fixtures.ts

```diff
@@ -121,6 +121,7 @@ base.beforeEach(async () => {
     const locator = page.locator('.notification-close-button').first()
     while (await locator.isVisible()) {
       await locator.click()
+      await page.waitForTimeout(100)
```

Comment:
`expect(Promise)...toXXX()` can handle this.

Reply:
Removed in https://github.com/logseq/logseq/pull/7538